index.bs

```diff
@@ -1153,12 +1159,6 @@ Benjamin Poulain for their contributions to the API specification.
 
 <pre class="biblio">
 {
-  "Models": {
-      "href": "https://github.com/webmachinelearning/webnn/blob/master/op_compatibility/first_wave_models.md",
-      "title": "The first-wave models",
-      "authors": ["Machine Learning for the Web Community Group"],
-      "date": "2020"
-  },
```

Comment:
Thanks @wonsuk73 for raising this issue. I also found it when rebasing #95 . My fix is to refer the models instead of deleting it, please take a look at https://github.com/webmachinelearning/webnn/pull/95/commits/ee9f4405e58479ee2c9ab2571da45d2a6ba7b8d0.

Reply:
@huningxin Thanks for your comment. Sorry that I didn't check your rebasing work^^; There is the reference for Models then we have to keep this.
But I don't know how can I bring this modification out from my PR because we use Bikteshed. Is there any solution to handle this?

Reply:
If you are fine the fix within #95, we can merge that PR first. Then you can rebase this PR and eliminate this change. Does it sound good to you?

Reply:
@huningxin I completely agree with your suggestion. But I don't know how can I rebase my PR for both files. Could you guide this? Thanks in advance!

Reply:
Sure. As #95 merged, you can rebase on top of the master branch. The index.bs should be fine. The index.html would have conflict, you can ignore that and finish the rebase. After that, you can regenerate index.html by running bikeshield. Then you may push the new index.bs and index.html into your branch. You may need `-f` to force update your branch as its base changes.

Reply:
My steps for your reference:
1. `git fetch origin`: fetch the latest origin/master
2. `git checkout <your_branch>`
3. `git rebase origin/master`: the index.html would have conflicts
4. `git restore --staged index.html`
5. `git restore index.html`: discard the index.html
6. `git restore --staged index.bs`
7. edit index.bs and revert the removing the models from biblio
8. `git add index.bs`
9. `git rebase --continue`
10. `make`: regenerate the index.html
11. `git add index.html`
12. `git commit`
13. `git push <your_remote> <your_branch>`

Reply:
@huningxin Thanks a lot for your very kind guide. I will try to do that with your guide! :)

Reply:
@huningxin When I push after rebase, I got error below. Is it correct?
remote: Permission to webmachinelearning/webnn.git denied to wonsuk73. fatal: unable to access 'https://github.com/webmachinelearning/webnn.git/': The requested URL ret
urned error: 403"

If this process is complicated, then I might withdraw this PR and re-do with new PR. I think this is simple way to do that :)

Reply:
> When I push after rebase, I got error below. Is it correct?

You need to push to your own forked repo (https://github.com/wonsuk73/webnn.git) and the branch (master) for this PR.

> If this process is complicated, then I might withdraw this PR and re-do with new PR. I think this is simple way to do that :)

This also works. Thanks!

Reply:
@huningxin Thanks for comment! I did push to my forked repo. It seems it works! Could you review this? :)

Reply:
Great. It looks good. Thanks.